max_min_icp_distance: 1.0
max_iterations: 10
leaf_size: 1.5